bitnami/concourse/templates/worker/deployment.yaml

```diff
@@ -0,0 +1,217 @@
+            {{- if .Values.worker.baggageclaim.bindIp }}
+            - name: CONCOURSE_BAGGAGECLAIM_BIND_IP
+              value: {{ .Values.worker.baggageclaim.bindIp | quote }}
+            {{- end }}
+            {{- if .Values.worker.baggageclaim.bindPort }}
+            - name: CONCOURSE_BAGGAGECLAIM_BIND_PORT
+              value: {{ .Values.worker.baggageclaim.bindPort | quote }}
+            {{- end }}
+            {{- if .Values.worker.baggageclaim.debugBindIp }}
+            - name: CONCOURSE_BAGGAGECLAIM_DEBUG_BIND_IP
+              value: {{ .Values.worker.baggageclaim.debugBindIp | quote }}
+            {{- end }}
+            {{- if .Values.worker.baggageclaim.debugBindPort }}
+            - name: CONCOURSE_BAGGAGECLAIM_DEBUG_BIND_PORT
+              value: {{ .Values.worker.baggageclaim.debugBindPort | quote }}
+            {{- end }}
+            {{- if .Values.worker.baggageclaim.volumes }}
+            - name: CONCOURSE_BAGGAGECLAIM_VOLUMES
+              value: {{ .Values.worker.baggageclaim.volumes | quote }}
+            {{- end }}
+            {{- if .Values.worker.baggageclaim.driver }}
+            - name: CONCOURSE_BAGGAGECLAIM_DRIVER
+              value: {{ .Values.worker.baggageclaim.driver | quote }}
+            {{- end }}
+            {{- if .Values.worker.baggageclaim.btrfsBin }}
+            - name: CONCOURSE_BAGGAGECLAIM_BTRFS_BIN
+              value: {{ .Values.worker.baggageclaim.btrfsBin | quote }}
+            {{- end }}
+            {{- if .Values.worker.baggageclaim.mkfsBin }}
+            - name: CONCOURSE_BAGGAGECLAIM_MKFS_BIN
+              value: {{ .Values.worker.baggageclaim.mkfsBin | quote }}
+            {{- end }}
+            {{- if .Values.worker.baggageclaim.overlaysDir }}
+            - name: CONCOURSE_BAGGAGECLAIM_OVERLAYS_DIR
+              value: {{ .Values.worker.baggageclaim.overlaysDir | quote }}
+            {{- end }}
+            {{- if .Values.worker.baggageclaim.disableUserNamespaces }}
+            - name: CONCOURSE_BAGGAGECLAIM_DISABLE_USER_NAMESPACES
+              value: {{ .Values.worker.baggageclaim.disableUserNamespaces | quote }}
+            {{- end }}
+          envFrom:
+            {{- if .Values.worker.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.worker.extraEnvVarsCM "context" $) }}
+            {{- end }}
+            {{- if .Values.worker.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.worker.extraEnvVarsSecret "context" $) }}
+            {{- end }}
+          {{- if .Values.worker.resources }}
+          resources: {{- toYaml .Values.worker.resources | nindent 12 }}
+          {{- end }}
+          {{- if .Values.worker.livenessProbe.enabled }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.worker.livenessProbe "enabled") "context" $) | nindent 12 }}
```

Comment:
This is incorrect in the template we have. Check solutions like Jupyterhub. Cannot use tplvalues.render here because if we change the probe it renders incorrect yaml

Reply:
Not yes done